public/js/script.js

```diff
@@ -6,4 +6,181 @@ for (let i = 0; i < createImageContainers.length; i++) {
        createImageContainers[i].style.backgroundImage = 'url(' + url + ')';
        createImageContainers[i].style.backgroundSize = 'cover';
    }
-}
\ No newline at end of file
+}
+
+window.addEventListener('load', () => {createEnableDisable(); createDisableAll();});
+window.addEventListener('keydown', () => {createEnableDisable();});
+window.addEventListener('click', () => {createEnableDisable();});
+
+function createDisableAll(){
+    enableDisableInputPrice(true);
+    enableDisableCharityConf(true);
+    enableDisableCharitySelector(true);
+    enableDisableInputExchange(true);
+    enableDisableCheckConf(true);
+}
+
+function countWords(){
+    let numWords = getDescription().value.trim().length;
+    if(numWords === 0){
+        getDescription().value = null;
+    }
+    document.getElementById("word_count")
+        .innerHTML = numWords + '/200';
+}
+
+//Toggles for create form
+function enableDisableButton(x){
+    document.getElementById('create_submit').disabled = x;
+
+}
+
+function enableDisableCharitySelector(x){
+    document.getElementById('charity-select').disabled = x;
+    if(x === true){
+        getCharityConf().checked = false;
+    }
+}
+
+function enableDisableCharityConf(x){
+    document.getElementById('checkCharity').disabled = x;
+    if(x === true){
+        getCharitySelector(). value = 'Select Charity';
+    }
+}
+
+function enableDisableInputPrice(x){
+    getInputPrice().disabled = x;
+    if(x === true){
+        getInputPrice().value = '';
+    }
+}
+
+function enableDisableInputExchange(x){
+    getInputExchange().disabled = x;
+    if(x === true){
+        getInputExchange().value = '';
+    }
+}
+
+function enableDisableCheckConf(x){
+    getCheckConf().disabled = x;
+    if(x === true){
+        getCheckConf().checked = false;
+    }
+}
+
+//Accessors for create
+
+function getInputPrice(){
+    return document.getElementById("inputPrice");
+}
+
+function getInputExchange(){
+    return document.getElementById('inputItem');
+}
+
+function getCheckConf(){
+    return document.getElementById('checkConf');
+}
+
+function getInputTitle(){
+    return document.getElementById('inputTitle');
+}
+
+function getDescription(){
+    return document.getElementById('description');
+}
+function getCharitySelector(){
+    return document.getElementById('charity-select')
+}
+function getCharityConf(){
+    return document.getElementById('checkCharity');
+}
+
+function getRadioSell(){
+    return document.getElementById('radio_sell');
+}
+function getRadioExchange(){
+    return document.getElementById('radio_exchange');
+}
+function getRadioGiveaway(){
+    return document.getElementById('radio_giveaway');
+}
```

Comment:
This is more subjective, but I wouldn't recommend using function like this, since it saves little time compared to document.getElementById. If you're writing them to avoid typos in string I'd recommend using a constant instead.